webapp/src/components/Variants/Edition/DraggableCommands/CommandListItem.tsx

```diff
@@ -145,6 +150,15 @@ function Item({ provided, item, style, isDragging, index, onDelete, onArgsUpdate
     setJsonData(e.updated_src);
     onArgsUpdate(index, e.updated_src);
   };
+
+  const onImport = (json: object) => {
+    if (checkCommandValidity(json)) {
```

Comment:
in the "else" condition, raise an error or display a error message box explaining that the json is not ok (though since it will be args in the json, there is, for now, no check to do here anymore.